packages/api/src/__tests__/useJurisdictionStatus.spec.tsx

```diff
@@ -0,0 +1,238 @@
+import * as React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import useAuthentication from '../hooks/useAuthentication';
+import usePOI from '../hooks/usePOI';
+import useJurisdictionStatus from '../hooks/useJurisdictionStatus';
+import APIProvider from '../APIProvider';
+
+jest.mock('@deriv/shared');
+jest.mock('../hooks/useAuthentication');
+jest.mock('../hooks/usePOI');
+
+const mockUseAuthentication = useAuthentication as jest.MockedFunction<typeof useAuthentication>;
+const mockUsePOI = usePOI as jest.MockedFunction<typeof usePOI>;
+
+describe('useJurisdictionStatus', () => {
+    test('for BVI/Labuan, should have a failed verification status if MT5 account status has failed and IDV status is rejected', () => {
+        const wrapper = ({ children }: { children: JSX.Element }) => <APIProvider>{children}</APIProvider>;
+        mockUsePOI.mockReturnValue({
+            // @ts-expect-error This is just for mocking, we just need to mock some properties only
+            data: {
```

Comment:
Ignored typescript checks for all object properties since this is just for mocking